File: ASP/API_CashMachine/Models/Machine.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace API_CashMachine.Models
{
    public class Machine
    {
        #region proriete

        private int _id;
        private int _maxCapacity;
        private int _thisCapacity;

        #endregion

        #region getter setter
        public int Id
        {
            get { return _id; }
            set { _id = value; }
        }
        public int MaxCapacity
        {
            get { return _maxCapacity; }
            set { _maxCapacity = value; }
        }
        public int ThisCapacity
        {
            get { return _thisCapacity; }
            set { _thisCapacity = value; }
        }
        #endregion

    }
}

File: ANDROID/p1cashv3/app/src/main/java/com/example/p1cashv3/models/Machine.java
package com.example.p1cashv3.models;

public class Machine {

    private int _id;
    private int _maxCapacity;
    private int _thisCapacity;

    public Machine(int _id, int _maxCapacity, int _thisCapacity) {
        this._id = _id;
        this._maxCapacity = _maxCapacity;
        this._thisCapacity = _thisCapacity;
    }

    public int get_id() {
        return _id;
    }
    public int get_maxCapacity() {
        return _maxCapacity;
    }
    public int get_thisCapacity() {
        return _thisCapacity;
    }

    public void set_thisCapacity(int _thisCapacity) {
        this._thisCapacity = _thisCapacity;
    }
}
